worlds/oot/__init__.py

```diff
@@ -1188,10 +1188,34 @@ def hint_type_players(hint_type: str) -> set:
 
     def fill_slot_data(self):
         self.collectible_flags_available.wait()
+        prizes = {}
+        bosses = {}
+        for location in [
+            'Links Pocket',
+            'Queen Gohma',
+            'King Dodongo',
+            'Barinade',
+            'Phantom Ganon',
+            'Volvagia',
+            'Morpha',
+            'Bongo Bongo',
+            'Twinrova',
+            # 'Ganon',
+        ]:
+            if not location == 'Links Pocket':
+                entrance = self.get_region(region_name=location+" Boss Room").entrances
+                entrance = entrance[0].name
+                dungeon = entrance[:str(entrance).index(" Boss Door")]
+            else:
+                dungeon = "Start"
+            prizes[self.get_location(location_name=location).item.name] = dungeon
```

Comment:
There's no need to specify the parameter and the older code doesn't either (this also applies elsewhere)
```suggestion
            prizes[self.get_location(location).item.name] = dungeon
```